exchanges/ftx/ftx_test.go

```diff
@@ -0,0 +1,1023 @@
+	if err == nil {
+		t.Error(err)
+	}
+}
+
+func TestGetFundingPayments(t *testing.T) {
+	t.Parallel()
+	if !areTestAPIKeysSet() {
+		t.Skip()
+	}
+	_, err := f.GetFundingPayments(time.Unix(1559881511, 0), time.Unix(1559901511, 0), "")
+	if err != nil {
+		t.Error(err)
+	}
+	_, err = f.GetFundingPayments(time.Unix(1559901511, 0), time.Unix(1559881511, 0), "")
+	if err == nil {
+		t.Error(err)
+	}
+}
+
+func TestListLeveragedTokens(t *testing.T) {
+	t.Parallel()
+	if !areTestAPIKeysSet() {
+		t.Skip()
+	}
+	_, err := f.ListLeveragedTokens()
+	if err != nil {
+		t.Error(err)
+	}
+}
+
+func TestGetTokenInfo(t *testing.T) {
+	t.Parallel()
+	if !areTestAPIKeysSet() {
+		t.Skip()
+	}
+	_, err := f.GetTokenInfo("")
+	if err != nil {
+		t.Error(err)
+	}
+}
+
+func TestListLTBalances(t *testing.T) {
+	t.Parallel()
+	if !areTestAPIKeysSet() {
+		t.Skip()
+	}
+	_, err := f.ListLTBalances()
+	if err != nil {
+		t.Error(err)
+	}
+}
+
+func TestListLTCreations(t *testing.T) {
+	t.Parallel()
+	if !areTestAPIKeysSet() {
+		t.Skip()
+	}
+	_, err := f.ListLTCreations()
+	if err != nil {
+		t.Error(err)
+	}
+}
+
+func TestRequestLTCreation(t *testing.T) {
+	t.Parallel()
+	if !areTestAPIKeysSet() {
+		t.Skip()
+	}
+	_, err := f.RequestLTCreation(testToken, 1)
+	if err != nil {
+		t.Error(err)
+	}
+}
+
+func TestListLTRedemptions(t *testing.T) {
+	t.Parallel()
+	if !areTestAPIKeysSet() {
+		t.Skip()
+	}
+	_, err := f.ListLTRedemptions()
+	if err != nil {
+		t.Error(err)
+	}
+}
+
+func TestGetQuoteRequests(t *testing.T) {
+	t.Parallel()
+	if !areTestAPIKeysSet() {
+		t.Skip()
+	}
+	_, err := f.GetQuoteRequests()
+	if err != nil {
+		t.Error(err)
+	}
+}
+
+func TestGetYourQuoteRequests(t *testing.T) {
+	t.Parallel()
+	if !areTestAPIKeysSet() {
+		t.Skip()
+	}
+	_, err := f.GetYourQuoteRequests()
+	if err != nil {
+		t.Error(err)
+	}
+}
+
+func TestCreateQuoteRequest(t *testing.T) {
+	t.Parallel()
+	if !areTestAPIKeysSet() {
+		t.Skip()
+	}
+	_, err := f.CreateQuoteRequest(strings.ToUpper(currency.BTC.String()), "call", order.Buy.Lower(), 1593140400, "", 10, 10, 5, 0, false)
+	if err != nil {
+		t.Error(err)
+	}
+}
+
+func TestDeleteQuote(t *testing.T) {
+	t.Parallel()
+	if !areTestAPIKeysSet() {
+		t.Skip()
+	}
+	_, err := f.DeleteQuote("1031")
+	if err != nil {
+		t.Error(err)
+	}
+}
+
+func TestGetQuotesForYourQuote(t *testing.T) {
+	t.Parallel()
+	if !areTestAPIKeysSet() {
+		t.Skip()
+	}
+	_, err := f.GetQuotesForYourQuote("1031")
+	if err != nil {
+		t.Error(err)
+	}
+}
+
+func TestMakeQuote(t *testing.T) {
+	t.Parallel()
+	if !areTestAPIKeysSet() {
+		t.Skip()
+	}
+	_, err := f.MakeQuote("1031", "5")
+	if err != nil {
+		t.Error(err)
+	}
+}
+
+func TestMyQuotes(t *testing.T) {
+	t.Parallel()
+	if !areTestAPIKeysSet() {
+		t.Skip()
+	}
+	_, err := f.MyQuotes()
+	if err != nil {
+		t.Error(err)
+	}
+}
+
+func TestDeleteMyQuote(t *testing.T) {
+	t.Parallel()
+	if !areTestAPIKeysSet() {
```

Comment:
Same for this one and please check for others down below, instead of mentioning each one